.pre-commit-config.yaml

```diff
@@ -9,15 +9,13 @@ repos:
         args:
           - --safe
           - --quiet
-        files: ^((components|esphome|script|tests)/.+)?[^/]+\.py$
```

Comment:
I don't think these are unnecessary. The are in the original file this was copied from https://github.com/esphome/esphome/blob/dev/.pre-commit-config.yaml
I typically run it with `pre-commit run --all-files`
Without them I think it would check all files in my .esphome and .venv directories. In fact I think all the new rules below have this problem. Do you have any suggestion how to fix this?

Reply:
I was concerned about this but I wasn't able to reproduce any problem here. Have you been able to? If I run:

```sh
esphome compile econet_heatpump_water_heater.yaml
pre-commit run --all-files
```

I don't get any failures. Maybe that's just lucky :-)

That said, it's easy to exclude the .esphome directory - do you think I should?